### ISSUE TITLE
mse: streams: EOF

### ISSUE THREAD

**Issue by scythk** [AlexxIT/WebRTC#772]
![image](https://github.com/user-attachments/assets/fd67c9c5-e8a6-4e9d-b768-54412d131980)
Both the RTSP stream directly from camera and the camera entity from ONVIF integration got "mse: streams: EOF" error sometimes, and both return to normal sometimes.

There are errors in homeassistant log:
Error from stream worker: Error opening stream (INVALIDDATA, Invalid data found when processing input, rtsp://****:****@192.168.**.**:554/stream1)

The camera entity itself works fine and the RTSP stream can be played using VLC player on my computer with no problem, so I assmue the problem is with the WebRTC integration.

**Comment by AlexxIT** [comment]
This error from core [stream](https://www.home-assistant.io/integrations/stream/) component.
```
Error from stream worker: Error opening stream (INVALIDDATA, Invalid data found when processing input, rtsp://:@192.168..:554/stream1)
```

Variants:
- you have bad camera
- your camera has bad connection (wi-fi, etc)
- your camera is overloaded

I can assume you have a TP-Link.

**Comment by scythk** [comment]
> This error from core [stream](https://www.home-assistant.io/integrations/stream/) component.
> 
> ```
> Error from stream worker: Error opening stream (INVALIDDATA, Invalid data found when processing input, rtsp://:@192.168..:554/stream1)
> ```
> 
> Variants:
> 
> * you have bad camera
> * your camera has bad connection (wi-fi, etc)
> * your camera is overloaded
> 
> I can assume you have a TP-Link.

Yes I am using TP-link, does TP-link camera have common issue like this? The TP-link app works good and the connection should be OK.
And what do you mean camera overloaded? Should I lower the bit rate or resoultion?

**Comment by AlexxIT** [comment]
If these cameras are used correctly - they works fine.
- they must have a good wifi connection
- they must have only one connection, maximum 2 (for example you already have minimum two - from stream integration and from this integration - not good)
- better to disable recording and motion detection on the camera

**Comment by scythk** [comment]
My older TP link camera works all good, but the newer one still has problem after using wired connection, lower resolution and turn off motion detection. Sometimes it got mse: streams: EOF and sometimes it got audio works but black screen with HLS or RTC on the upper right corner.
Had to switch back to the older one for now.
